user/templates/windows/windows-qubes-4-1.md

```diff
@@ -0,0 +1,274 @@
+     ~~~
+     powercfg -H off
+     ~~~
+    Also, recent versions of Windows won’t show the CD-ROM drive after starting the qube with `qvm-start vm --cdrom ...` (or using the GUI). The solution is to disable hibernation in Windows with this command. (That command is included in QWT’s setup but it’s necessary to run it manually in order to be able to open QWT’s setup ISO/CD-ROM in Windows).
+
+  - In case you switch from `sys-firewall` to `sys-whonix`, you'll need a static IP network configuration, DHCP won't work for `sys-whonix`. Sometimes this may also happen if you keep using `sys-firewall`. In both cases, proceed as follows:
+    - Check the IP address allocated to the qube - either from GUI Manager, or via `qvm-ls -n WindowsNew` from a dom0 terminal (E.g. 10.137.0.x with gateway 10.138.y.z).
+    - In the Windows qube, open the Network manager and change the IPv4 configuration of the network interfacefrom "Automatic" to "Manual".
+      - Enter the Address: 10.137.0.x in our example.
+      - Enter the Netmask: 255.255.255.0
+      - Enter the Gateway: 10.138.y.z in our example.
+      - Enter DNS: 10.139.1.1,10.139.1.2 (the Virtual DNS addresses used by Qubes.
+    - Click "Apply". You should now see "Connected".
+
+  - Given the higher than usual memory requirements of Windows, you may get a `Not enough memory to start domain 'WindowsNew'` error. In that case try to shutdown unneeded VMs to free memory before starting the Windows VM.
+
+    At this point you may open a tab in dom0 for debugging, in case something goes amiss:
+
+     ~~~
+     tailf /var/log/qubes/vm-WindowsNew.log \
+        /var/log/xen/console/hypervisor.log \
+        /var/log/xen/console/guest-WindowsNew-dm.log
+     ~~~
+
+At that point you should have a functional and stable Windows VM, although without updates, Xen's PV drivers nor Qubes integration (see sections [Windows Update](/doc/templates/windows/windows-qubes-4-1/#windows-update) and [Xen PV drivers and Qubes Windows Tools](/doc/templates/windows/qubes-windows-tools-4-1/#xen-pv-drivers-and-qubes-windows-tools)). It is a good time to clone the VM again.
+
+**Installing Qubes Windows Tools**
+
+To install Qubes Windows Tools, follow instructions in [Qubes Windows Tools](/doc/templates/windows/qubes-windows-tools-4-1), but don’t forget to `qvm-clone` your qube before you install Qubes Windows Tools (QWT) in case something goes south.
+
+**Post-install best practices**
+
+Optimize resources for use in virtual machine as “vanilla” version of Windows are bloated; e.g.:
+
+- optionally use `disk cleanup` to save some disk space.
+- set up Windows for best performance (this pc → advanced settings → …)
+- think about Windows’ page file: is it needed ? should you set it with a fixed size ? maybe on the private volume ?
+- investigate “debloat” scripts ; eg. Windows10Debloater 2, decrapifier, OOShutUp10 1, etc.
```

Comment:
I don't think we should endorse various 3rd-party tools in our official documentation, especially in section "best practices". This _at least_ need some disclaimer that user should evaluate risks themselves, but probably better remove this point.